pkg/apis/core/types.go

```diff
@@ -1434,11 +1434,11 @@ type KeyToPath struct {
 	Mode *int32
 }
 
-// Local represents directly-attached storage with node affinity
+// Local represents directly-attached storage with node affinity (Beta feature)
 type LocalVolumeSource struct {
-	// The full path to the volume on the node
-	// For alpha, this path must be a directory
-	// Once block as a source is supported, then this path can point to a block device
+	// The full path to the volume on the node, it can be a a disk, partition or directory.
+	// When you use the local volume as a raw block device, you must set the volumes with
```

Comment:
"Directories can be represented only by PersistentVolume with VolumeMode=Filesystem. Block devices can use VolumeMode=Block only".

Can I use a block device as VolumeMode=Filesystem? IMO it's not possible right now.